releases/release-1.22/release-notes/maps/pr-101542-map.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 101542
+releasenote:
+  text: Scheduler's CycleState now embeds internal read/write locking inside its Read()
+    and Write() functions. Meanwhile, Lock() and Unlock() function are removed.                 scheduler
```

Comment:
may Write(), Read(), Lock() and  Unlock() should be in code blocks as they are methods. Also, there is a lot of white spaces and Scheduler should be with a capital letter.

Reply:
Done